usr/share/lib/img_proof/tests/SLES/test_sles_wait_on_registration.py

```diff
@@ -8,9 +8,14 @@ def test_sles_wait_on_registration(host):
 
     while time.time() < end:
         result = host.run('systemctl is-active guestregister.service')
+        status = result.stdout.strip()
 
-        if result.stdout.strip() == 'inactive':
+        if status == 'inactive':
```

Comment:
Consider doing this as a guard clause.

Reply:
It basically is a guard clause though? Just happens inside the loop to prevent code duplication. But the sleep call only happens after the first two clauses.